src/draw/handler/Draw.Feature.js

```diff
@@ -13,13 +13,14 @@ L.Draw.Feature = L.Handler.extend({
 		if (options && options.shapeOptions) {
 			options.shapeOptions = L.Util.extend({}, this.options.shapeOptions, options.shapeOptions);
 		}
-		L.setOptions(this, options);
+
+		L.ToolbarAction.prototype.initialize.call(this, options);
```

Comment:
This is tightly coupling the draw handlers to the toolbar :( The handlers need to be able to be used with or without the leaflet.draw controls. This enables people to only include one handler (or multiple) and do the manual activation as per usual Leaflet handlers.